diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0962c40..1171f97 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -409,7 +409,6 @@ if( ${_OPT}has_vst3 AND CMAKE_CXX_COMPILER_ID MATCHES "GNU" )
    #VST3 module_linux.cpp uses c++ fs features
    #Once compiler minimum version is upgraded to 9.1+
    #we don't need this option any more
-   list( APPEND CMAKE_REQUIRED_LIBRARIES -lstdc++fs )
 endif()
 
 # Add the math library (if found) to the list of required libraries
--- audacity/libraries/lib-vst3/module_linux.cpp-orig	2023-04-15 10:49:45.143347408 -0400
+++ audacity/libraries/lib-vst3/module_linux.cpp	2023-04-15 10:49:57.590564371 -0400
@@ -43,7 +43,7 @@
 #include <sys/utsname.h>
 #include <unistd.h>
 
-#if (__cplusplus >= 201707L)
+#if (__cplusplus >= 201703L)
 #if __has_include(<filesystem>)
 #define USE_EXPERIMENTAL_FS 0
 #elif __has_include(<experimental/filesystem>)
