diff --git a/libraries/lib-vst/VSTEffectsModule.cpp b/libraries/lib-vst/VSTEffectsModule.cpp
index 88f11746d..b2c22ab94 100644
--- a/libraries/lib-vst/VSTEffectsModule.cpp
+++ b/libraries/lib-vst/VSTEffectsModule.cpp
@@ -238,8 +238,8 @@ PluginPaths VSTEffectsModule::FindModulePaths(PluginManagerInterface & pm)
       pathList.push_back(wxT(LIBDIR) wxT("/vst"));
 
       // These are the defaults used by other hosts
-      pathList.push_back(wxT("/usr/lib/vst"));
-      pathList.push_back(wxT("/usr/local/lib/vst"));
+      // Flatpak default
+      pathList.push_back(wxT("/app/extensions/Plugins/vst"));
       pathList.push_back(wxGetHomeDir() + wxFILE_SEP_PATH + wxT(".vst"));
    }
 
