diff --git a/src/effects/VST3/VST3EffectsModule.cpp b/src/effects/VST3/VST3EffectsModule.cpp
index 162464d9e..87c14bc0b 100644
--- a/src/effects/VST3/VST3EffectsModule.cpp
+++ b/src/effects/VST3/VST3EffectsModule.cpp
@@ -172,7 +172,7 @@ VST3EffectsModule::FindModulePaths(PluginManagerInterface &)
 #elif __WXGTK__
    pathList.push_back(wxGetHomeDir() + "/.vst3/");
    pathList.push_back("/usr/lib/vst3/");
-   pathList.push_back("/usr/local/lib/vst3/");
+   pathList.push_back("/app/extensions/Plugins/vst3/");
 #endif
 
    // bundled/app specific
diff --git a/src/effects/VST3/module_linux.cpp b/src/effects/VST3/module_linux.cpp
index 80e2c30d6..e7f8c2051 100644
--- a/src/effects/VST3/module_linux.cpp
+++ b/src/effects/VST3/module_linux.cpp
@@ -43,7 +43,7 @@
 #include <sys/utsname.h>
 #include <unistd.h>
 
-#if (__cplusplus >= 201707L)
+#if (__cplusplus >= 201703L)
 #if __has_include(<filesystem>)
 #define USE_EXPERIMENTAL_FS 0
 #elif __has_include(<experimental/filesystem>)
