diff --git a/cmake-proxies/lv2/CMakeLists.txt b/cmake-proxies/lv2/CMakeLists.txt
index 289688eb9..f998a0805 100644
--- a/cmake-proxies/lv2/CMakeLists.txt
+++ b/cmake-proxies/lv2/CMakeLists.txt
@@ -201,7 +201,7 @@ elseif( UNIX )
    set( LILV_DEFAULT_LV2_PATH "~/.lv2:/usr/lib/lv2:/usr/local/lib/lv2" )
 
    set( SUIL_MODULE_DIR "" )
-   set( SUIL_DIR_SEP "" )
+   set( SUIL_DIR_SEP "/" )
    set( SUIL_GTK2_LIB_NAME "libgtk-x11-2.0.so.0" )
    set( SUIL_GTK3_LIB_NAME "libgtk-x11-3.0.so.0" )
    set( SUIL_MODULE_PREFIX "lib" )
--- a/src/CMakeLists.txt	2021-03-18 22:59:02.577128541 -0400
+++ b/src/CMakeLists.txt	2021-03-18 22:59:29.260332383 -0400
@@ -1073,7 +1073,7 @@
 
 set( BUILDING_AUDACITY YES )
 set( INSTALL_PREFIX "${_PREFIX}" )
-set( PKGLIBDIR "${_PKGLIBDIR}" )
+set( PKGLIBDIR "${_PKGLIB}" )
 set( LIBDIR "${CMAKE_INSTALL_FULL_LIBDIR}" )
 set( HAVE_GTK ${GTK_FOUND} )
 
