diff --git a/cmake-proxies/CMakeLists.txt b/cmake-proxies/CMakeLists.txt
index 5de68d0c6..ea238a068 100644
--- a/cmake-proxies/CMakeLists.txt
+++ b/cmake-proxies/CMakeLists.txt
@@ -113,14 +113,18 @@ else()
    set ( curl_ssl "openssl" )
 endif ()
 
+if( ${_OPT}has_networking )
 add_conan_lib(
    ThreadPool
    threadpool/20140926
    REQUIRED
    ALWAYS_ALLOW_CONAN_FALLBACK
 )
-
-if( ${_OPT}has_networking )
+add_conan_lib(
+   RapidJSON
+   rapidjson/1.1.0
+   REQUIRED
+)
 
    add_conan_lib(
       CURL
@@ -148,11 +152,6 @@ if( NOT CMAKE_SYSTEM_NAME MATCHES "Darwin|Windows")
    )
 endif()
 
-add_conan_lib(
-   RapidJSON
-   rapidjson/1.1.0
-   REQUIRED
-)
 
 set_conan_vars_to_parent()
 
